### PR TITLE
EclipseFoundation.TheiaBlueprint version 1.16.0

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26097)

### CODE REVIEW

File: manifests/e/EclipseFoundation/TheiaBlueprint/1.16.0/EclipseFoundation.TheiaBlueprint.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: EclipseFoundation.TheiaBlueprint
+PackageVersion: 1.16.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://mirrors.tuna.tsinghua.edu.cn/eclipse/theia/latest/windows/TheiaBlueprint.exe
+  InstallerSha256: 921D41CE6623AD8A4A5B774D8C62A3A67E9B462384C83DBD5A62E41345FDE957
```
Comment (OfficialEsco) [inline manifests/e/EclipseFoundation/TheiaBlueprint/1.16.0/EclipseFoundation.TheiaBlueprint.installer.yaml:10]:
```suggestion
- Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://mirrors.tuna.tsinghua.edu.cn/eclipse/theia/latest/windows/TheiaBlueprint.exe
  InstallerSha256: 921D41CE6623AD8A4A5B774D8C62A3A67E9B462384C83DBD5A62E41345FDE957
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://mirrors.tuna.tsinghua.edu.cn/eclipse/theia/latest/windows/TheiaBlueprint.exe
  InstallerSha256: 921D41CE6623AD8A4A5B774D8C62A3A67E9B462384C83DBD5A62E41345FDE957
  InstallerSwitches:
    Custom: /ALLUSERS
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — DISMISSED** [review]

**Comment by jamespik** [comment]
This appears to be pointing to a mirror rather than the official download link on the project website.

**Comment by oxygen-dioxide** [comment]
ok

**Comment by oxygen-dioxide** [comment]
ok

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**